src/components/ActionButtons/BragButton.js

```diff
@@ -12,10 +12,18 @@ const StyledButton = styled(ButtonBase)`
   border-bottom-color: ${colorDarkGreen};
 `;
 
+const TEXT = 'Brag about my efforts';
```

Comment:
Maybe something a bit more explicit would be better like BUTTON_LABEL
Also new line separating both const is not required imho.